oc-chef-pedant/spec/api/account/account_acl_spec.rb

```diff
@@ -1142,29 +1162,72 @@ def rand_id
             end # context GET /<type>/<name>/_acl/<permission>
 
             context "PUT /#{type}/<name>/_acl/#{permission}" do
-
+              let(:clients) { [platform.non_admin_client.name] }
+              let(:users) {
+                  [platform.non_admin_user.name, platform.admin_user.name, "pivotal"]
+              }
               let(:groups_and_actors) {{
                   "actors" => [platform.non_admin_user.name,
-                    platform.admin_user.name, "pivotal"].uniq,
+                    platform.admin_user.name, "pivotal"].uniq + clients,
                   "groups" => ["admins", "users", "clients"]
                 }}
               let(:update_body) {{
                   permission => groups_and_actors
                 }}
 
-              context "admin user" do
-                it "can update ACL" do
-                  put(permission_request_url, platform.admin_user,
-                    :payload => update_body).should look_like({
-                      :status => 200
-                    })
-                  check_body = acl_body
-                  check_body[permission] = groups_and_actors
+              context "admin user"  do
+                context "using the new 'users' and 'clients' attributes" do
+                  let(:update_body) {
+                    { permission => {
+                      "actors" => [], # back-compat, empty actors and
+                                      # clients/users present indicates
+                                      # that clients/users should be used.
```

Comment:
There's a test for that [here](https://github.com/chef/chef-server/pull/912/files#diff-1fa1ea176b4030c87671fad2db6220e2R1346), it will also 400.